webpack.config.js

```diff
@@ -169,11 +152,17 @@ const entryHtmlPlugins = Object.keys(entry).map(function(entryName) {
     if (entryName == 'config-admin') {
         inject = 'head'
         fileName = 'admin'
-    }
+    }    
+    const templateFile = htmlTemplatesFiles.find((file) => {
```

Comment:
i do think that here would be more quicker if u check if file belongs to the default entry points
on defaultEntryPoints